.gitmodules

```diff
@@ -78,10 +78,7 @@
     ignore = dirty
     path = third_party/fmt
     url = https://github.com/fmtlib/fmt.git
-[submodule "third_party/tensorpipe"]
     ignore = dirty
```

Comment:
@pragupta nit: we should also remove this line, it's already present for the previous entry of fmt.git